lib/enum/nested.rb

```diff
@@ -18,21 +18,27 @@ def initialize(people_array)
   end
 
   def ages_sum
+    @people.reduce(0) { |sum,human| sum += human.age }
   end
 
   def average_age
+    @people.reduce(0) { |sum,human| sum += human.age } / @people.size
   end
 
   def total_years_programming_experience_for_all_languages
+    people.map(&:years_language_experience).reduce(0) { |sum, language| sum += language[1] }
   end
 
   def favorite_food_frequency
+    people.map {&:favorite_foods }.flatten.each_with_object(Hash.new(0)) { |sum, hash| hash[sum]+=1 }
```

Comment:
You've got a syntax error here that's causing your tests to not run. Which makes me wonder how often you're running your tests. Remember, you should be running your tests after every single method you write.